cmd/minikube/cmd/start.go

```diff
@@ -1779,3 +1781,61 @@ func exitGuestProvision(err error) {
 	}
 	exit.Error(reason.GuestProvision, "error provisioning guest", err)
 }
+
+func startNerdctld() {
+	// create a temporary directory
+	tmpDirName, err := os.MkdirTemp("/tmp", "minikube-nerdctld")
```

Comment:
lets move all the download logic to a new func called

downloadNerdCtl()

and call that func, later once this feature is released, we could have a follow up PR to remove that func and just add them to the base imagfe